pkg/app/piped/cmd/piped/piped.go

```diff
@@ -170,6 +170,24 @@ func (p *piped) run(ctx context.Context, input cli.Input) (runErr error) {
 		}
 	}
 
+	// Login to OCI registry.
+	if regs := cfg.OCIHelmChartRegistries(); len(regs) > 0 {
```

Comment:
how about `regs := cfg.HelmChartRegistries()` to parse list of all chart registries (not just OCI typed chart) and try login to them all (use switch/case blog and call to `loginToOCIRegistry()` function in case of reg of type OCI) 🤔

Reply:
Agree!

Reply:
I implement this! : [ee22536](https://github.com/pipe-cd/pipecd/pull/3643/commits/ee22536d72768310d8afaed36ab612ee415142f0)